video/out/vo_tct.c

```diff
@@ -160,7 +159,7 @@ static void write_plain(bstr *frame,
             if (buffering <= VO_TCT_BUFFER_PIXEL)
                 print_buffer(frame);
         }
-        bstr_xappend(NULL, frame, TERM_ESC_CLEAR_COLORS);
+        bstr_xappend(NULL, frame, (bstr)bstr0_lit(TERM_ESC_CLEAR_COLORS));
```

Comment:
Maybe we should add `bstr_xappend0` in `bstr.h`, I've seen it used in other places too.